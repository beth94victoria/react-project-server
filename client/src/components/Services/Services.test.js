import React from 'react'
import renderer from 'react-test-renderer'
import Services from './Services'

describe('Services snapshot', () => {
    it('Should render as expected', () => {
        const tree = renderer.create(
        <Services page="http://localhost:3000/services">Services</Services>
        )
        expect(tree.toJSON()).toMatchSnapshot()
    })
})