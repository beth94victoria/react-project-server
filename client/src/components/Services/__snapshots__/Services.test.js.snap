// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services snapshot Should render as expected 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-6 align-items-center"
      >
        <div>
          <img
            alt="mower"
            className="rounded servimg"
            src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408306/Five%20Star/mower_res.jpg"
          />
          <br />
          <p
            className="servicetext"
          >
            LAWN MOWING
          </p>
        </div>
      </div>
      <div
        className="col-6 align-items-center"
      >
        <div>
          <img
            alt="whipsnip"
            className="rounded servimg"
            src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408307/Five%20Star/whippersnipper_res.jpg"
          />
          <br />
          <p
            className="servicetext"
          >
            WHIPPER SNIPPING
          </p>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-6"
      >
        <img
          alt="weeding"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408307/Five%20Star/weeding_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          WEEDING
        </p>
      </div>
      <div
        className="col-6"
      >
        <img
          alt="hedging"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408306/Five%20Star/hedging_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          HEDGE TRIMMING
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-6"
      >
        <img
          alt="spraying"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408307/Five%20Star/spraying_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          PLANT SPRAYING
        </p>
      </div>
      <div
        className="col-6"
      >
        <img
          alt="rubbish"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408306/Five%20Star/rubbish_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          RUBBISH REMOVAL
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-6"
      >
        <img
          alt="gardening"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408306/Five%20Star/gardening_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          GARDENING
        </p>
      </div>
      <div
        className="col-6"
      >
        <img
          alt="treetrim"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408307/Five%20Star/treetrim_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          TREE TRIMMING
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-6"
      >
        <img
          alt="treecut"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408307/Five%20Star/treeremove_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          TREE REMOVAL
        </p>
      </div>
      <div
        className="col-6"
      >
        <img
          alt="turfing"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408307/Five%20Star/turf_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          TURF LAYING
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-6"
      >
        <img
          alt="fertilizer"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408306/Five%20Star/fert_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          FERTILISE LAWNS
        </p>
      </div>
      <div
        className="col-6"
      >
        <img
          alt="pressurewash"
          className="rounded servimg"
          src="https://res.cloudinary.com/dvqlfwgtf/image/upload/v1549408306/Five%20Star/pressurewash_res.jpg"
        />
        <br />
        <p
          className="servicetext"
        >
          PRESSURE WASHING
        </p>
      </div>
    </div>
  </div>
</div>
`;
